iRecruit-ui/src/components/FixedSidebar/FixedSidebar.jsx

```diff
@@ -1,8 +1,7 @@
-
-import React from "react";
+import React, { useEffect, useState } from "react";
 import "./FixedSidebar.css";
 
-const FixedSidebar = () => {
+const FixedSidebar = ({ posts, userInteractions }) => {
   // Random data for the bar chart for app status to test the UI
   const shortlistedCount = Math.floor(Math.random() * 15);
   const pendingCount = Math.floor(Math.random() * 20);
```

Comment:
Are the sizes of the colors on the bar proportional to the count in relation to each other?

Reply:
Yes they are.